src/index.js

```diff
@@ -19,6 +27,9 @@ class RpService {
 
     const {id} = await promise;
     process.env.RP_LAUNCH_ID = id;
+    if (this.options.writeLaunchIdToFile) {
```

Comment:
would be better to make not just boolean option, but path to `RP_LAUNCH_ID`
 file

Reply:
but it is not required

Reply:
Done! If `true` is supplied it will default to `RP_LAUNCH_ID`.